src/test-utils/test-utils.ts

```diff
@@ -110,6 +111,9 @@ export const getSession: jest.MockedFunction<GetSession> = jest.fn(() => ({
     eventCount: 0
 }));
 
+export const sourceMapsFetchFunction: jest.MockedFunction<SourceMapsFetchFunction> =
```

Comment:
Backlog item: this needs an integration test to validate that source maps are being applied correctly

Reply:
Yes, I stoped development and shared the PR so that RUM could decide if this client solution is something they want to move forward with vs. a service side solution or maybe this is a temporary solution? If we want to move forward I will work on the integration tests.